src/mlpack/methods/ann/rnn_impl.hpp

```diff
@@ -364,18 +367,20 @@ EvaluateWithGradient(const arma::mat& /* parameters */,
           std::move(arma::mat(responses.slice(0).colptr(begin),
           responses.n_rows, batchSize, false, true)), std::move(error));
     }
+    size_t x = 0;
+    x = (rho>r_size)?r_size:rho;    
```

Comment:
style issue `x = (rho > r_size) ? r_size : rho`;

Again may be we could avoid introducing the variable